core/domain/story_domain.py

```diff
@@ -168,14 +170,12 @@ def __init__(
         Args:
             node_id: str. The unique id for each node.
             title: str. The title of the story node.
+            description: str. The description for the story node.
             thumbnail_filename: str|None. The thumbnail filename of the story
-                node.
```

Comment:
Why are this and lines 174 and 178 deleted? Revert it. Again, make sure you do a manual review of your code yourself before pushing.

Reply:
I am sorry these changes happened without my knowledge when I was resolving merge conflicts. And since these are in comments section even the program didn't flag them.
Done.